test/tools/CreateChildProcess/CreateChildProcess.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Diagnostics;
+using System.Threading;
+
+namespace CreateChildProcess
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            if (args.Length > 0)
+            {
+                uint num = UInt32.Parse(args[0]);
+                for (uint i = 0; i < num; i++)
+                {
+                    Process child = new Process();
+                    child.StartInfo.FileName = Process.GetCurrentProcess().MainModule.FileName;
+                    child.Start();
+                }
+            }
+            Thread.Sleep(100000);
```

Comment:
I thought it was better to have it "timeout" rather than hang if the process wasn't successfully terminated.  Maybe I should just leave it to the CI system to kill if it takes too long, but may be better to have this test fail and allow other tests to run within the CI time limit.